test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature

```diff
@@ -0,0 +1,537 @@
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+Feature: delete an attribute request using NGSI v2 API. "DELETE" - /v2/entities/<entity_id>/attrs/<attr_name>
+  As a context broker user
+  I would like to delete an attribute request using NGSI v2 API
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  Actions After each Scenario:
+  Setup: delete database in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+  @happy_path
+  Scenario: Delete an attribute in several entities using NGSI v2 API
+    Given a definition of headers
+      | parameter          | value                  |
+      | Fiware-Service     | test_delete_happy_path |
+      | Fiware-ServicePath | /test                  |
+      | Content-Type       | application/json       |
+    And create "3" entities with "3" attributes
+      | parameter        | value       |
+      | entities_type    | house       |
+      | entities_id      | room        |
+      | attributes_name  | temperature |
+      | attributes_value | 45          |
+      | attributes_type  | date        |
+      | metadatas_number | 2           |
+      | metadatas_name   | very_hot    |
+      | metadatas_type   | alarm       |
+      | metadatas_value  | hot         |
+    And verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+    When delete an attribute "temperature_0" in entities with id "room"
+      | parameter       | value                                  |
+      | entities_number | the same value of the previous request |
+    Then verify that receive several "No Content" http code
+    And verify that the attribute is deleted into mongo
```

Comment:
If the test case is about removing the attribute in the 3 entities in a single operation, this maybe should be more explicit, e.g.:

"And verify that the attribute is deleted into mongo in the "3" entities"

or something similar